server/src/main/java/io/spine/server/entity/storage/ColumnType.java

```diff
@@ -101,7 +101,7 @@
      * <p>Used if the actual value of the {@link EntityColumn},
      * defined for storing, is {@code null}.
      *
-     * @param storageRecord the database record
+     * @param storageRecord    the database record
      * @param columnIdentifier the identifier of the column, e.g. its index
```

Comment:
The method params in this file trigger warnings, saying that in the descendants there is no `@ParametersAreNonnullByDefault` defined in their `package-info.java`. I found that in some of `.given` packages there is no `package-info.java` at all. 

See, in `/test/java` sources most of packages are the same as in `/main/java`. And in `/main/java` we have package info files for every package, and IDE "counts" those as the description for every file in the package, including the test sources.

However in `/test/java` there are `... .given` packages, which are unique for the test codebase. And we have to add `package-info.java` files for every such package. I can see that in `server` module not every `... .given` package has this file. Could you please add those, including some description + `@CheckReturnValue`and `@ParametersAreNonnullByDefault`annotations.